lib/modules/manager/helmfile/types.ts

```diff
@@ -7,5 +7,6 @@ export interface Doc {
   repositories?: {
```

Comment:
see other comment, maybe you need to extract a new sub type

Reply:
Done !